src/main/java/uk/gov/digital/ho/hocs/document/JSONToSimpleTextConverter.java

```diff
@@ -55,6 +55,7 @@ private void buildString(JsonNode node, String keyName, int level) {
         } else {
             String textValue = node.textValue();
             if (keyName.equals("complaintText")) {
+                textValue = textValue.replaceAll("[\\n]", NEW_LINE_STR);
```

Comment:
Worth adding some tests for this?

Reply:
Updated existing test covering building formatted document